### ISSUE TITLE
for Python 3 distinction

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/ipaddr-py#82]
```
This is just a feature request for the webpage, i got her using google and i'm 
wondering what version is the latest stable for Python 3, i couldn't find it on 
the readme and the changelog is not clear about it, i'm a little afraid of 
installing the library and messing up my Python 3 installation, i'm kinda new 
with Python and i'd love to use this library since i just spent half morning 
creating my own parser and manipulator for IP's :P

TLDR. which version should i download for Python 3?


```

Original issue reported on code.google.com by `guillerm...@gmail.com` on 5 Mar 2011 at 7:37
